app/controllers/idv/capture_doc_controller.rb

```diff
@@ -12,6 +12,14 @@ class CaptureDocController < ApplicationController
       analytics_id: Analytics::CAPTURE_DOC,
     }.freeze
 
+    def show
```

Comment:
We should be able to do this by adding a new step that mirrors the document capture step and adding that to the capture doc flow.

Reply:
> We should be able to do this by adding a new step that mirrors the document capture step and adding that to the capture doc flow.

Would this essentially be the copy-paste approach from the original comment?

>Copy paste `views/idv/doc_auth/document_capture.html.erb` to `views/idv/capture_doc/document_capture.html.erb`

Reply:
> We should be able to do this by adding a new step that mirrors the document capture step and adding that to the capture doc flow.

In rebased b5f1334ea68533771810bbf9b83022da279d3482, we now have a shared partial `idv/shared/_document_capture.html.erb` rendered by both `idv/doc_auth/document_capture.html.erb` and `idv/capture_doc/document_capture.html.erb`.